apps/bfd-pipeline/bfd-pipeline-app/src/main/java/gov/cms/bfd/pipeline/app/PipelineApplication.java

```diff
@@ -321,6 +321,7 @@ private static boolean anySmokeTestFailed(List<PipelineJob<?>> jobs) {
    * Create all pipeline jobs and return them in a list.
    *
    * @param appConfig our {@link AppConfiguration} for configuring jobs
+   * @param appMeters the app meters
```

Comment:
no link?

Reply:
I don't typically make lines on the params, just when doing the getter/setter descriptions. (Existing ones I left as-is)